deploy.desc

```diff
@@ -7,4 +7,4 @@ SECRETS=(SPLITIO_API_KEY SPLITIO_EXT_API_KEY)
 CPU_REQUEST="5m"
 MEMORY_REQUEST="85Mi"
 NGINX_CPU_REQUEST="15m"
-NGINX_MEMORY_REQUEST="195Mi"
+NGINX_MEMORY_REQUEST="770Mi"
```

Comment:
550Mi is probably fine, I think the 770 you were seeing in gcloud console was the sum of both containers.

Reply:
Just updated the image and I could be interpreting the graph incorrectly but it looks like it's just taking the average. My thought was to update the memory request so that the `(usage / requests) * 100` was ~50. Let me know if there's a better number to shoot for.

Reply:
true, 50% might be a good number to shoot for when updating these.  That way we don't have to come back to them.